OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -1217,7 +1349,10 @@ public static DreamValue NativeProc_regex(DreamObject instance, DreamObject usr,
             int sides;
             int modifier = 0;
             if (arguments.ArgumentCount == 1) {
-                string diceInput = arguments.GetArgument(0, "ndice").GetValueAsString();
+                if(!arguments.GetArgument(0, "ndice").TryGetValueAsString(out var diceInput))
+                {
+                    return new DreamValue(0);
```

Comment:
`roll(null)` gives `1`, though the 2-arg variant of this proc gives `0`
```suggestion
                    return new DreamValue(1);
```